stable/kommander/requirements.lock

```diff
@@ -4,13 +4,13 @@ dependencies:
   version: 0.1.9
 - name: yakcl-federation
   repository: https://mesosphere.github.io/yakcl/charts
-  version: 0.0.3
+  version: v0.0.4
 - name: yakcl-licensing
   repository: https://mesosphere.github.io/yakcl/charts
-  version: 0.0.3
+  version: v0.0.4
 - name: yakcl-provisioning
   repository: https://mesosphere.github.io/yakcl/charts
-  version: 0.0.3
+  version: v0.0.4
```

Comment:
Is it `v0.0.4` or `0.0.4`?

Reply:
This is what helm does for me since the early days